xla/stream_executor/device_description.proto

```diff
@@ -44,9 +44,11 @@ message GpuDeviceInfoProto {
   float clock_rate_ghz = 13;
   int64 device_memory_size = 14;
   int32 shared_memory_per_block_optin = 15;
+  int64 registers_per_core_limit = 16;
```

Comment:
Don't change the proto numbers. See https://protobuf.dev/programming-guides/dos-donts/#reuse-number. Instead assign the new fields numbers 18 and 19.